subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/fixtures/ToolingApiSpec.groovy

```diff
@@ -174,6 +173,7 @@ trait ToolingApiSpec {
             args.remove("--no-daemon")
 
             model = connection.model(type)
+                .forTasks(tasks)
```

Comment:
What was the reason for this change 9why wasn;t it needed before)? Beware I have had little contact with the Tooling API.

Reply:
Previously we haven't testing scenarios for model building for tasks in ToolingApiSpec. We tested only model building. This change is allowing to include tasks execution as well. If we pass `tasks==null` then no tasks will be executed.